daliuge-engine/dlg/apps/branch.py

```diff
@@ -16,8 +16,8 @@
 # @param input_error_threshold 0/Integer/ComponentParameter/NoPort/ReadWrite//False/False/the allowed failure rate of the inputs (in percent), before this component goes to ERROR state and is not executed
 # @param n_tries 1/Integer/ComponentParameter/NoPort/ReadWrite//False/False/Specifies the number of times the 'run' method will be executed before finally giving up
 # @param dummy_input /Object/ApplicationArgument/InputPort/ReadWrite//False/False/Dummy input port
-# @param dummy0 /Object/ApplicationArgument/OutputPort/ReadWrite//False/False/Dummy output port
-# @param dummy1 /Object/ApplicationArgument/OutputPort/ReadWrite//False/False/Dummy output port
+# @param yes /Object/ApplicationArgument/OutputPort/ReadWrite//False/False/True condition output port
```

Comment:
Was there any reason we didn't name them True/False? I'm not familiar with the `branch` construct so am likely missing something.

Reply:
I think "true" and "false" are probably the best names.

I made the change to "yes" and "no" to match what the EAGLE graph checker looks for.

But if there is no specific reason to use "yes" and "no", then I prefer "true" and "false".

Reply:
After talking to @awicenec , we'll use "true" and "false".